### PR TITLE
Update readme, option to open dashboard

### PR DESCRIPTION
**Pull request by mikael10j**
Fix README.

Add arg parsing to open_dashboard to open a custom one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ It is a working prototype used in CTA, especially by the [GammaLearn](https://gi
 ## Install
 
 ```
-pip install gammaboard
+cd gammaboard
```
Comment (vuillaut) [inline README.md:24]: that should be exactly the same?

File: gammaboard/gammaboard.py [hunk 2]
```diff
@@ -852,9 +853,17 @@ def open_dashboard():
...
 
     """
-    original_dashboard_path = find_resource('dashboard.ipynb')
-    with tempfile.TemporaryDirectory() as tmpdir:
-        tmp_dashboard = os.path.join(tmpdir, 'dashboard.ipynb')
-        copyfile(original_dashboard_path, tmp_dashboard)
-        command = 'jupyter notebook {}'.format(tmp_dashboard)
-        os.system(command)
\ No newline at end of file
+    parser = argparse.ArgumentParser()
```
Comment (vuillaut) [inline gammaboard/gammaboard.py]: This is not the intended behavior.
The user should not have to worry where is his dashboard file. He should not even have to care that there is a file...
Eventually, I could imagine a use case where you want to have an optionnal argument to launch an alternative dashboard. But in this case, I think you don't need a custom `gammaboard` command, you may simply use `jupyter notebook`...

Comment (mikael10j) [reply gammaboard/gammaboard.py]: indeed !

File: gammaboard/gammaboard.py [hunk 3]
```diff
@@ -857,4 +857,4 @@ def open_dashboard():
         tmp_dashboard = os.path.join(tmpdir, 'dashboard.ipynb')
         copyfile(original_dashboard_path, tmp_dashboard)
         command = 'jupyter notebook {}'.format(tmp_dashboard)
-        os.system(command)
\ No newline at end of file
+    os.system(command)
```
Comment (vuillaut) [inline gammaboard/gammaboard.py]: Nop, the command must be executed in the while, otherwise, the temp directory does not exist anymore.

**Review by vuillaut — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mikael10j) [reply README.md:24]: `pip install gammaboard` needs the package to be uploded to pypi.

Comment (mikael10j) [reply README.md:24]: > (torch12) mikael@lappc-p730:~/gammaboard$ pip install gammaboard
Collecting gammaboard
  ERROR: Could not find a version that satisfies the requirement gammaboard (from versions: none)
ERROR: No matching distribution found for gammaboard

File: gammaboard/gammaboard.py (hunk 3 above)
Comment (mikael10j) [reply gammaboard/gammaboard.py]: just forgot to indent

File: README.md (hunk 1 above)
Comment (vuillaut) [reply README.md:24]: Ha yes, the subtle difference between `pip install gammaboard` and `pip install gammaboard/` 😅

Comment (vuillaut) [reply README.md:24]: the version with cd is less error prone.

**Review by vuillaut — APPROVED** [review]

**Pull request merged**